src/foam/nanos/google/api/sheets/import/GoogleSheetsImportConfig.js

```diff
@@ -0,0 +1,110 @@
+foam.CLASS({
+  package: 'foam.nanos.google.api.sheets',
+  name: 'GoogleSheetsImportConfig',
+  documentation: 'GoogleSheetsImportConfig contains info about import',
+  properties: [
+    {
+      name: 'importClassInfo',
+      class: 'Class',
+      javaType: 'foam.core.ClassInfo',
+      hidden: true,
+    },
+    {
+      name: 'googleSheetLink',
+      class: 'String',
+      view: {
+        class: 'foam.u2.TextField',
+      },
+      label: 'Google Sheets Link',
+      postSet: function() {
+        var findMatch = this.googleSheetLink.match(this.SPREADSHEET_ID_REGEX);
+       this.googleSpreadsheetId = findMatch ? findMatch[1]: findMatch;
+      },
+      required: true
+    },
+    {
+      name: 'cellsRange',
+      class: 'String',
+      hidden: true
+    },
+    {
+      name: 'googleSpreadsheetId',
+      class: 'String',
+      view: {
+        class: 'foam.u2.TextField',
+      },
+      label: 'Google Spreadsheet Id',
+      hidden: true,
+      expression: function(googleSheetLink) {
+        var findMatch = googleSheetLink.match(this.SPREADSHEET_ID_REGEX);
+        return  findMatch ? findMatch[1] : findMatch;
+      }
+    },
+    {
+      name: 'googleSheetId',
+      class: 'String',
+      view: {
```

Comment:
You could just do: view: 'foam.u2.TextField'.

Reply:
fixed